zetaclient/chains/base/observer.go

```diff
@@ -236,6 +242,12 @@ func (ob *Observer) WithLastTxScanned(txHash string) *Observer {
 	ob.mu.Lock()
 	defer ob.mu.Unlock()
 
+	if ob.lastTxScanned == "" {
+		ob.logger.Chain.Debug().
```

Comment:
Since initialization is a one time log print, showing where it scans from with `Info` level is also good?